MetaPhase-Consulting/State-TalentMAP
# Cache network requests as fallback

Cache some network responses (the ones that populate the filters) and only fall back to them if there are subsequent network failures to the same request URL.
- Use package.json version to prevent conflicting data structures when we push new versions of the app

Test by:
- Open Chrome Dev Tools
- Load Search Results page to completion
- Network tab -> right click a filter-related network request -> block request URL
- Refresh the page
- The network request should show red for failure, but the filter will still load since it is using the cached version
